addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -337,6 +359,16 @@ var ListRenderer = BasicRenderer.extend({
         }
         this.arch.children = children;
     },
+    /**
+     * This method tells if current session is ran by a user who will be able to
+     * use or install odoo studio.
+     * 
+     * @private
+     * @returns {boolean}
+     */
+    _isAllowedToUseStudio: function(){
+        return session.is_admin || session.is_system;
+    },
```

Comment:
Functional specs says user must have group system, maybe should I only verify session.is_system (see : https://github.com/odoo/odoo/blob/536133ef6db84079f1bb8ba424e0e3b80533adaa/odoo/addons/base/models/res_users.py#L872-L878)